### PR TITLE
A fragmenter that returns a list of files instead of splits

### PR DESCRIPTION
**Pull request by frankgh**
A fragmenter that returns a list of files instead of splits

### CODE REVIEW

**Comment by shivzone** [comment]
I guess this is intended to be a WIP PR as opposed to a PR

**Comment by shivzone** [comment]
I don't think we need a separate fragmenter for this as this would bloat the # of profiles. If we go with a user defined property, we can add this functionality in existing fragmenters which can support this

File: server/pxf-hdfs/src/main/java/org/greenplum/pxf/plugins/hdfs/HdfsDataFragmenter.java [hunk 1]
```diff
@@ -65,18 +65,19 @@ public void initialize(RequestContext context) {
     public List<Fragment> getFragments() throws Exception {
         Path path = new Path(hcfsType.getDataUri(configuration, context));
         List<InputSplit> splits = getSplits(path);
+        String[] hosts = {"foo"};
```
Comment (shivzone) [inline server/pxf-hdfs/src/main/java/org/greenplum/pxf/plugins/hdfs/HdfsDataFragmenter.java]: We should just set it to null. FragmentsResponseFormatters.convertHostsToIPs handles null hosts appropriately

**Comment by frankgh** [comment]
Closing in favor of https://github.com/greenplum-db/pxf/pull/158

**Pull request closed without merging**